form/src/components/Form.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import Question from "./Question";
+import Result from "./Result";
+
+const Form = () => {
+  const [answer, setAnswer] = useState(" ");
+  const [isSubmit, setIsSubmit] = useState(false);
+  const handleCheckInput = (e) => {
+    setAnswer(e.target.value);
+  };
+  const handleClick = (e) => {
+    e.preventDefault();
+    if (
+      answer.toLowerCase().trim() === "yes" ||
+      answer.toLowerCase().trim() === "no"
+    ) {
+      alert(`question : ${answer.trim()} submitted`);
```

Comment:
use `errorMessage` variable to display error message to user in UI (ıt might be below input field or end of form). Not in alert prompt. Plus, you should satisfy the condition: `Display an error message ("You didn't ask a question"), if user didn't write "?"(question mark) in the input`